.gitignore

```diff
@@ -1 +1,3 @@
-.idea/
\ No newline at end of file
+.idea/
+hooks/.dart_tool
+hooks/.packages
```

Comment:
```suggestion
```

Reply:
I don’t think those are necessary since you added them to the hooks .gitignore

Reply:
> .DS_Store

That is odd, I don't see it in my system. But I added to the project's gitignore now.